tagcompare/compare.json

```diff
@@ -16,6 +16,9 @@
         "browserName": "chrome"
       }
     },
+    "phantomjs": {
+      "enabled": true
+    },
```

Comment:
Yeah it's not exactly clear because `compare.json` really does 2 things: 1. list all the set of browser configs and 2. list the comparisons you'd wanna do against the configs

(1) doesn't really need to be configurable, is your suggestion to make it part of the code, in say `browserconfigs.py`?
(2) does need to be configurable because the runner of tagcompare should be able to specify locally what kind of comparisons he cares about

Out of scope for this PR though - opened https://github.com/paperg/tagcompare/issues/54